### ISSUE TITLE
Laggy scrolling with AttributedTextView inside tableViewCell

### ISSUE THREAD

**Issue by daurenm** [evermeer/AttributedTextView#16]
Hi there,

Seems like AttributedTextView.attributer.setter takes too much time while blocking main thread. 

<img width="858" alt="screen shot 2018-03-31 at 12 49 29 pm" src="https://user-images.githubusercontent.com/13380740/38160572-032ad1be-34e2-11e8-94ea-5efd19b2a587.png">

Is there anything we can do to make it faster? Seems like right now, it requires main thread, so I can't simply put in a background thread :(

**Comment by evermeer** [comment]
Hi,

Yes, you ar right, It now assumes you are doing everything on the main. 
I think that for now you could use a workaround like this:
```
DispatchQueue.global().async {
    let at: NSMutableAttributedString = "red".red.attributedText
    DispatchQueue.main.async {
        self.myLabel.attributedText = at
    }
}
```
I will think about how to embed functionality like this into the library.

**Comment by evermeer** [comment]
@jameshays Good point. I was already hesitating implementing it the way I suggested.
